web/source/keyboards/kmwkeyboards.ts

```diff
@@ -664,7 +662,8 @@ namespace com.keyman.keyboards {
               manager.keymanweb.domManager._SetTargDir(manager.keymanweb.domManager.getLastActiveElement());
             }
 
-            String.kmwEnableSupplementaryPlane(kbd && ((kbd['KS'] && kbd['KS'] == 1) || kbd['KN'] == 'Hieroglyphic')); // I3319 - SMP extension, I3363 (Build 301)
+            String.kmwEnableSupplementaryPlane(true);
```

Comment:
Do we need this if it is being done in keymanweb.ts during init?

Reply:
Same response as before.